apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -321,6 +321,9 @@ static void tc_libc_timer_strftime(void)
 	st_time.tm_mday = 19;
 	st_time.tm_mon = 5;
 	st_time.tm_year = 2017 - TM_YEAR_BASE;
+#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)
+	st_time.tm_wday = 0;
```

Comment:
How about matching this value based on the above?

Reply:
I agree with your opinion.
I will change the value of st_time.tm_wday to 5 (friday)